### PR TITLE
Re-arranging troubleshooting info

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/u96Pr1vO/91-bio-formats-general-doc-improvements
Troubleshooting page was buried and contained random selection of info better placed elsewhere.

### CODE REVIEW

**Comment by gusferguson** [comment]
@hflynn 

Reads well to me. 
Can't check layout but markup looks all fine.
Good to merge.

**Comment by hflynn** [comment]
You can check the layout on the staging pages - https://www.openmicroscopy.org/site/support/bio-formats5.1-staging/
I already spotted a formatting issue with that annoying purple highlighting now, I'll push a fix

**Comment by hflynn** [comment]
Changes to the formats pages weren't built due to some uncaught error in the autogen build. Hopefully this will be fixed tomorrow when this will need checking for correct layout.

File: components/autogen/src/format-pages.txt [hunk 1]
```diff
@@ -517,6 +519,10 @@ reader = DicomReader.java
 notes = * DICOM stands for "Digital Imaging and Communication in Medicine". \n
 * Bio-Formats supports both compressed and uncompressed DICOM files. \n
 \n
+If you have a problematic DICOM file which you cannot send us for privacy \n
+reasons, please send us the exact error message and be aware that it may take \n
+several attempts to fix the problem blind.
```
Comment (sbesson) [inline components/autogen/src/format-pages.txt]: Missing `\n` at the end of this line

**Comment by jburel** [comment]
Changes look good other than @sbesson comments

**Comment by hflynn** [comment]
Hopefully http://ci.openmicroscopy.org/view/Docs/job/BIOFORMATS-5.1-merge-docs-autogen/ will be green tomorrow now!

**Comment by sbesson** [comment]
https://ci.openmicroscopy.org/view/Docs/job/BIOFORMATS-5.1-merge-docs-autogen/158/ :+1:

**Comment by hflynn** [comment]
yay!

**Comment by jburel** [comment]
nitpick:  ```an SVS or JPEG-2000 file``` should be ```a```

**Comment by hflynn** [comment]
@mtbc & I agree it is correct as 'an' SVS file, as does http://openslide.org/formats/aperio/ (see TIFF Image Directory Organization section)

**Comment by jburel** [comment]
ok, you will have to explain to me why it is correct from a grammatical point of view, since I am not a native speaker.
Should ```a JPEG-2000 or SVS file``` be an?

**Comment by hflynn** [comment]
If JPEG was first, it would be 'a'. 'An' is right for SVS because it is pronounced as 'ess' i.e. as if it begins with a vowel, and the rules for 'a' vs. 'an' are based on how it sounds spoken.

**Comment by jburel** [comment]
@hflynn: thanks for the clarification. I will keep that in mind

**Comment by jburel** [comment]
ready to merge

**Comment by hflynn** [comment]
I think this would probably rebase ok, we should probably decide if we are going to rebase all the docs improvements for 5.0.7 or not bother.

**Comment by hflynn** [comment]
Actually, give this removes a url completely, now I've thought about it some more, probably better not to rebase and just leave all the docs updates as 5.1 only.

--no-rebase

**Pull request merged**